apps/backend/src/users/users.controller.ts

```diff
@@ -10,4 +10,12 @@ export class UsersController {
   getAllUsers() {
     return this.usersService.findAll();
   }
+
+  @Put(':userId')
```

Comment:
:pick: should this be a [patch](https://stackoverflow.com/questions/24241893/should-i-use-patch-or-put-in-my-rest-api) route?